pkgs/os-specific/linux/e1000e/default.nix

```diff
@@ -4,19 +4,19 @@ assert stdenv.lib.versionOlder kernel.version "4.10";
 
 stdenv.mkDerivation rec {
   name = "e1000e-${version}-${kernel.version}";
-  version = "3.3.5.3";
+  version = "3.8.4";
 
   src = fetchurl {
     url = "mirror://sourceforge/e1000/e1000e-${version}.tar.gz";
-    sha256 = "1ajz3vdnf1y307k585w95r6jlh4ah8d74bq36gdkjl1z5hgiqi9q";
+    sha256 = "1q8dbqh14c7r15q6k6iv5k0d6xpi74i71d5r54py60gr099m2ha4";
   };
 
   hardeningDisable = [ "pic" ];
 
   configurePhase = ''
     cd src
     kernel_version=${kernel.modDirVersion}
-    sed -i -e 's|/lib/modules|${kernel.dev}/lib/modules|' Makefile
+    sed -i -e 's|/lib/modules|${kernel.dev}/lib/modules|' Makefile common.mk
```

Comment:
`/lib/modules` is now only used in `Makefile` for the `rpm` and `uninstall` targets. So, only `common.mk` needs to be updated. Also, for single file substitutions `substituteInPlace` is preferred, so it would be best to replace this `sed` command by:

```
    substituteInPlace common.mk \
      --replace "/lib/modules" "${kernel.dev}/lib/modules"
```

Reply:
Right. Thanks for making me aware, I will amend and push shortly!

Reply:
I included your changes in 587bb6eace2. Please resolve this conversation if you like the new version.